test/e2e/e2e.go

```diff
@@ -254,7 +321,7 @@ func PodLogs(clients *test.Clients, podName string, containerName string, namesp
 			}).Do()
 			raw, err := result.Raw()
 			if err == nil {
-				logger.Infof("%s logs request result: %#v", podName, string(raw))
+				logger.Debugf("%s logs request result: %#v", podName, string(raw))
```

Comment:
I've actually found these very useful, why downgrade them?

Reply:
It is somewhat messy when you send a bunch of events, and we are logging them in the caller. 
But I'll rollback the change and leave it as Info